src/cli/near.rs

```diff
@@ -119,6 +132,36 @@ pub enum ReadCommand {
 
 #[derive(Subcommand)]
 pub enum WriteCommand {
+    /// Deploy and initialize a new instance of the Aurora Engine.
+    /// Uses the `engine_account_id` from the config as the target account.
+    /// `config.near_key_path` must point to a full access key for `engine_account_id`.
+    EngineInit {
+        /// Path to the Wasm artifact for the engine contract.
+        #[clap(short, long)]
+        wasm_path: String,
+        #[clap(short, long)]
+        chain_id: u64,
```

Comment:
We should have documents for each of these, even if its self explanatory.

Reply:
Additional documentation added in https://github.com/aurora-is-near/aurora-cli-rs/pull/8/commits/0765289b1f9de35ce6a6d1ef9e4228f9de38459d